### DISCUSSION TITLE
Search results are always empty

### DISCUSSION THREAD

**Discussion by danyaridiger** [Androz2091/discord-player#1804] [Q&A] [answered]
I am trying to search video from Youtube and extract `tracks` field from search results, but since last few updates this field is always empty. My code looks like:

`const result = await player.search(url, {
    requestedBy: interaction.user,
    searchEngine: QueryType.YOUTUBE_VIDEO,
  });
`

When `url` is either url of Youtube video or string query. Previously i could retrieve `tracks` from this result but now it doesn't contain it in any case. May be i am using deprecated way of searching?

**Comment by twlite** [comment]
> I am trying to search video from Youtube and extract `tracks` field from search results, but since last few updates this field is always empty. My code looks like:
> 
> `const result = await player.search(url, { requestedBy: interaction.user, searchEngine: QueryType.YOUTUBE_VIDEO, }); `
> 
> When `url` is either url of Youtube video or string query. Previously i could retrieve `tracks` from this result but now it doesn't contain it in any case. May be i am using deprecated way of searching?

Do you have the extractors loaded?

**Reply by danyaridiger** [reply]
No, i didn't. How can i do it?

**Comment by twlite** [comment, marked as answer]
> No, i didn't. How can i do it?

- Make sure you have `@discord-player/extractor` installed
- add this code after declaration of Player: `player.extractors.loadDefault()`

**Reply by danyaridiger** [reply]
Thanks! After adding this line search works like previously.
